server/gfmd/user.h

```diff
@@ -70,3 +70,8 @@ gfarm_error_t gfm_server_user_info_remove(struct peer *, int, int);
 struct group_assignment;
 /* subroutine of grpassign_add(), shouldn't be called from elsewhere */
 void grpassign_add_group(struct group_assignment *);
+
+struct user_auth;
+gfarm_error_t user_auth_id_moidfy(struct user *, const char *,
```

Comment:
there is a typo here (due to my fault).
Please replace user_auth_id_moidfy by user_auth_id_modify.